cli/bpmetadata/cmd.go

```diff
@@ -308,7 +308,12 @@ func CreateBlueprintDisplayMetadata(bpPath string, bpDisp, bpCore *BlueprintMeta
 	bpDisp.Spec.Info.Source = bpCore.Spec.Info.Source
 	buildUIInputFromVariables(bpCore.Spec.Interfaces.Variables, bpDisp.Spec.Ui.Input)
 
-	existingInput := proto.Clone(bpCore.Spec.Ui.Input).(*BlueprintUIInput)
+	existingInput := func() *BlueprintUIInput {
```

Comment:
Could we add a test for this case too?

Reply:
let me add one in follow up PR.

Reply:
This is in cmd.go which I don't think we have unit testing coverage. Do we want to create unit tests for this file? Also @qz267 do we already have some other tests for the display metadata generation?

Reply:
I would prefer we merge this first and do the refactoring later so we can also try extract other logics from cmd.go that need unit testing. Created a bug for this.

Reply:
@qz267 shall we include this as part of our testing improvement plan?